cristin-organization/src/test/java/no/unit/nva/cristin/organization/fetch/FetchCristinOrganizationHandlerTest.java

```diff
@@ -242,6 +224,31 @@ void shouldReturnOrganizationFlatHierarchy() throws IOException {
         assertThat(actualOrganization.getPartOf(), equalTo(null));
     }
 
+    @Test
+    @Disabled
+    void shouldMapUpstreamJsonCorrectlyForVersionTwo() throws Exception {
+        var resource = stringFromResources(CRISTIN_GET_RESPONSE_V2_JSON);
+        var fakeHttpResponse = new HttpResponseFaker(resource, HTTP_OK);
+        doReturn(fakeHttpResponse).when(apiClient20230526)
+            .fetchGetResult(URI.create("https://api.cristin-test.uio.no/v2/units/1.2.3.4"));
+
+        var subsResource = stringFromResources(CRISTIN_GET_RESPONSE_V2_WITH_SUBS_JSON);
+        var fakeSubsHttpResponse = new HttpResponseFaker(subsResource, HTTP_OK);
+        doReturn(fakeSubsHttpResponse).when(apiClient20230526)
+            .fetchGetResult(URI.create("https://api.cristin-test.uio.no/v2/units?parent_unit_id=1.2.3.4&per_page=2000"));
+
+        fetchCristinOrganizationHandler = new FetchCristinOrganizationHandler(clientProvider, new Environment());
+        fetchCristinOrganizationHandler.handleRequest(requestUsingVersionTwo(), output, context);
+
+        var gatewayResponse = GatewayResponse.fromOutputStream(output, Organization.class);
+        var responseBody = gatewayResponse.getBodyObject(Organization.class);
+
+        var pretty = OBJECT_MAPPER.readTree(responseBody.toString()).toPrettyString();
+
+        System.out.println(pretty);
```

Comment:
use logger instead

Reply:
Sorry, work in progress. will fix before asking for new review